test/compress/drop-unused.js

```diff
@@ -177,3 +177,58 @@ keep_fnames: {
         }
     }
 }
+
+assign_only: {
+    options = { unused: true };
+    input: {
+        function f1() {
+            var a;
+            a = 1;
+        }
+
```

Comment:
Please remove the blank lines between the functions in this test so it's easier to compare the input to the output. Thanks.